### PR TITLE
Decrease bloat + bug fixes

### PR DESCRIPTION
**Pull request by w4ft**
- Changed how windows OS is detected, now using os.name()
- Removed redundant checks for exe files, this is now only checked once in the beginning
- Added clear command that runs correct command for clearing the terminal for OS
- Fixed bug in bulk function where log file is displayed before it is checked to exist
- Removed Bulk1 function for videos. Combined with bulk() with added filetype parameter
- **Removed need to copy/paste binary in new working directory**
- Fixed bug where working directory is not reset after run, so next run has the wrong working directory

**_The run.bat batch script needs to be changed so that "py MRT" is "py %~dp0MRT" This makes sure its finding MRT in the current working directory_**

### CODE REVIEW

**Comment by Anish-M-code** [comment]
@w4ft please remove binary files like pycache from git commits

**Comment by Anish-M-code** [comment]
@w4ft it seems you have forgotten to remove all ds store files . Maybe adding ds store files to gitignore might be a good idea.

File: Source Code/.DS_Store
Comment (Anish-M-code) [inline Source Code/.DS_Store:1]: Ds store to be removed

File: Source Code/Python Source Code/.DS_Store
Comment (Anish-M-code) [inline Source Code/Python Source Code/.DS_Store:1]: Ds store to be removed

File: Source Code/Python Source Code/MRT/.DS_Store
Comment (Anish-M-code) [inline Source Code/Python Source Code/MRT/.DS_Store:1]: Ds store to be removed

**Comment by Anish-M-code** [comment]
the code looks fine , after a manual test will approve 👍

**Comment by w4ft** [comment]
@Anish-M-code removed ds store!

**Comment by w4ft** [comment]
@Anish-M-code Does everything look good?

**Comment by evil-user** [comment]
Sorry for the delay @w4ft @Anish-M-code  due to laptop issues , i couldn't test this PR but i am marking it as high priority and will test it by next sunday.

**Comment by Anish-M-code** [comment]
Everything working fine. Once again thank you @wft for your contribution 👍

**Pull request merged**